fuzzers/libafl-fuzz/src/mutational_stage.rs

```diff
@@ -0,0 +1,116 @@
+use std::{borrow::Cow, marker::PhantomData};
+
+use libafl::{
+    inputs::Input,
+    mutators::Mutator,
+    stages::{mutational::MutatedTransform, MutationalStage, Stage},
+    state::{HasCorpus, HasRand, State, UsesState},
+    Error, Evaluator, HasNamedMetadata,
+};
+use libafl_bolts::Named;
+
+#[derive(Debug)]
+pub enum SupportMutationalStages<S, SM, P, E, EM, M, I, Z> {
```

Comment:
do you mean Support_ed_ stages?

Reply:
something like `EnabledMutationalStage::...`?

Reply:
I would rather supported. Yes, typo!